android/src/main/java/com/getcapacitor/community/intercom/intercom/IntercomPlugin.java

```diff
@@ -272,6 +276,17 @@ private void setUpIntercom() {
             String apiKey = config.getPluginConfiguration("Intercom").getString("androidApiKey");
             String appId = config.getPluginConfiguration("Intercom").getString("androidAppId");
 
+            switch (IntercomPushManager.getInstalledModuleType()) {
```

Comment:
not sure about this part as I can't test. but do we really need to tell Intercom to cache a sender id? if so, something to investigate and move to the app layer as well.